delivery-metrics/sql/create_delivery_metrics_db.sql

```diff
@@ -0,0 +1,105 @@
+# drop tables
+
+DROP TABLE IF EXISTS deliverable;
+DROP TABLE IF EXISTS deliverable_quad_map;
+DROP TABLE IF EXISTS epic;
+DROP TABLE IF EXISTS epic_deliverable_map;
+DROP TABLE IF EXISTS issue;
+DROP TABLE IF EXISTS issue_history;
+DROP TABLE IF EXISTS issue_sprint_map;
+DROP TABLE IF EXISTS sprint;
+DROP TABLE IF EXISTS quad;
+
+# create tables
+
+CREATE TABLE deliverable (
+	id INTEGER PRIMARY KEY AUTOINCREMENT,
+	guid TEXT UNIQUE NOT NULL,
+	title TEXT NOT NULL,
+	pillar TEXT, 
+	t_created TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+	t_modified TIMESTAMP 
+);
+
+CREATE TABLE deliverable_quad_map (
+	id INTEGER PRIMARY KEY AUTOINCREMENT,
+	deliverable_id INTEGER NOT NULL,
+	quad_id INTEGER,
+	d_effective DATE NOT NULL,
+	t_modified TIMESTAMP,
+	UNIQUE(deliverable_id, d_effective)
+);
+
+CREATE TABLE epic (
+	id INTEGER PRIMARY KEY AUTOINCREMENT,
+	guid TEXT UNIQUE NOT NULL,
+	title TEXT NOT NULL,
+	t_created TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+	t_modified TIMESTAMP 
+);
+
+CREATE TABLE epic_deliverable_map (
+	id INTEGER PRIMARY KEY AUTOINCREMENT,
+	epic_id INTEGER NOT NULL,
+	deliverable_id INTEGER,
+	d_effective DATE NOT NULL,
+	t_modified TIMESTAMP,
+	UNIQUE(epic_id, d_effective)
+);
+
+CREATE TABLE issue (
+	id INTEGER PRIMARY KEY AUTOINCREMENT,
+	guid TEXT UNIQUE NOT NULL,
+	title TEXT NOT NULL,
+	type TEXT NOT NULL,
+	opened_date DATE,
+	closed_date DATE,
+	parent_issue_guid TEXT,
+	epic_id INTEGER,
+	t_created TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+	t_modified TIMESTAMP 
+);
+
+CREATE TABLE issue_history (
+	id INTEGER PRIMARY KEY AUTOINCREMENT,
+	issue_id INTEGER NOT NULL,
+	status TEXT NOT NULL,
```

Comment:
_Technically_ status can be NULL. It _shouldn't_ be for 99% of issues if the workflow automation on projects are set up correctly, but someone can manually unset the status field on an issue and I wouldn't want that to halt the insert.

```suggestion
	status TEXT,
```

Reply:
Good catch. Fixed in [2b16f4b](https://github.com/agilesix/simpler-grants-sandbox/pull/48/commits/2b16f4bdf1c3f10ae05af214cba81f7a086df3bb)